components/notifications.js

```diff
@@ -251,6 +251,39 @@ function Reply ({ n }) {
   )
 }
 
+function NotificationAlert () {
+  const [showAlert, setShowAlert] = useState(false)
+  const pushNotify = useNotification()
+
+  useEffect(() => {
+    setShowAlert(!localStorage.getItem('hideNotifyPrompt'))
+  }, [])
+
+  const close = () => {
+    localStorage.setItem('hideNotifyPrompt', 'yep')
+    setShowAlert(false)
+  }
+
+  return (
+    showAlert
+      ? (
+        <Alert variant='success' dismissible onClose={close}>
+          Enable push notifications?
+          <button
+            className='btn mx-1'
+            onClick={() => {
+              pushNotify.requestPermission()
+              close()
+            }}
+          >Yes
+          </button>
+          <button className='btn mx-1' onClick={close}>No</button>
```

Comment:
Added `mx-1` because of https://web.dev/accessible-tap-targets/

Else there is no cursor change when going between "Yes" and "No". 

(Btw, I noticed the same with bounty posts in the past. There was no intermediate cursor change between "reply" and "pay bounty".)